2023-03-17 18:12:39,967 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Configure stats pid to 83239
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Loading settings from /home/tharikh/.config/wandb/settings
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Loading settings from /home/tharikh/dl-ass-1/wandb/settings
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-17 18:12:39,968 INFO    MainThread:83239 [wandb_init.py:_log_setup():492] Logging user logs to /home/tharikh/dl-ass-1/wandb/run-20230317_181239-pzgfpo53/logs/debug.log
2023-03-17 18:12:39,969 INFO    MainThread:83239 [wandb_init.py:_log_setup():493] Logging internal logs to /home/tharikh/dl-ass-1/wandb/run-20230317_181239-pzgfpo53/logs/debug-internal.log
2023-03-17 18:12:39,969 INFO    MainThread:83239 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9a0af77be0>
2023-03-17 18:12:39,969 INFO    MainThread:83239 [wandb_init.py:init():532] calling init triggers
2023-03-17 18:12:39,969 INFO    MainThread:83239 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-17 18:12:39,969 INFO    MainThread:83239 [wandb_init.py:init():554] re-initializing run, found existing run on stack: qhfwea90
2023-03-17 18:12:39,970 INFO    MainThread:83239 [wandb_run.py:_finish():1854] finishing run cs22m058/dl-project-final/qhfwea90
2023-03-17 18:12:39,970 INFO    MainThread:83239 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-17 18:12:39,970 INFO    MainThread:83239 [wandb_run.py:_restore():2066] restore
2023-03-17 18:12:39,971 INFO    MainThread:83239 [wandb_run.py:_restore():2072] restore done
2023-03-17 18:12:45,475 INFO    MainThread:83239 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-17 18:12:45,475 INFO    MainThread:83239 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-17 18:12:45,479 INFO    MainThread:83239 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-17 18:12:45,485 INFO    MainThread:83239 [wandb_init.py:init():588] starting backend
2023-03-17 18:12:45,486 INFO    MainThread:83239 [wandb_init.py:init():592] setting up manager
2023-03-17 18:12:45,489 INFO    MainThread:83239 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-17 18:12:45,491 INFO    MainThread:83239 [wandb_init.py:init():599] backend started and connected
2023-03-17 18:12:45,519 INFO    MainThread:83239 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-17 18:12:45,519 INFO    MainThread:83239 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-17 18:12:45,519 INFO    MainThread:83239 [wandb_init.py:init():687] updated telemetry
2023-03-17 18:12:45,537 INFO    MainThread:83239 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-17 18:12:47,447 INFO    MainThread:83239 [wandb_run.py:_on_init():2134] communicating current version
2023-03-17 18:12:47,812 INFO    MainThread:83239 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-17 18:12:47,812 INFO    MainThread:83239 [wandb_init.py:init():775] starting run threads in backend
2023-03-17 18:12:57,711 INFO    MainThread:83239 [wandb_run.py:_console_start():2114] atexit reg
2023-03-17 18:12:57,711 INFO    MainThread:83239 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-17 18:12:57,712 INFO    MainThread:83239 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-17 18:12:57,712 INFO    MainThread:83239 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-17 18:12:57,712 INFO    MainThread:83239 [wandb_init.py:init():817] run started, returning control to user process
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Configure stats pid to 83239
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Loading settings from /home/tharikh/.config/wandb/settings
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Loading settings from /home/tharikh/dl-ass-1/wandb/settings
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_init.py:_log_setup():492] Logging user logs to /home/tharikh/dl-ass-1/wandb/run-20230317_181257-bmd6189g/logs/debug.log
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_init.py:_log_setup():493] Logging internal logs to /home/tharikh/dl-ass-1/wandb/run-20230317_181257-bmd6189g/logs/debug-internal.log
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_init.py:init():532] calling init triggers
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-17 18:12:57,758 INFO    MainThread:83239 [wandb_init.py:init():554] re-initializing run, found existing run on stack: pzgfpo53
2023-03-17 18:12:57,760 INFO    MainThread:83239 [wandb_run.py:_finish():1854] finishing run cs22m058/dl-project-final/pzgfpo53
2023-03-17 18:12:57,760 INFO    MainThread:83239 [jupyter.py:save_history():449] not saving jupyter history
2023-03-17 18:12:57,760 INFO    MainThread:83239 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-17 18:12:57,760 INFO    MainThread:83239 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-17 18:12:57,760 INFO    MainThread:83239 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-17 18:12:57,760 INFO    MainThread:83239 [wandb_run.py:_restore():2066] restore
2023-03-17 18:12:57,760 INFO    MainThread:83239 [wandb_run.py:_restore():2072] restore done
2023-03-17 18:13:10,937 INFO    MainThread:83239 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-17 18:13:10,938 INFO    MainThread:83239 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-17 18:13:10,948 INFO    MainThread:83239 [wandb_run.py:_footer_sync_info():3398] logging synced files
